app/src/main/java/com/internship/tvseries/data/repository/config/ConfigRetrofitRepository.java

```diff
@@ -0,0 +1,43 @@
+package com.internship.tvseries.data.repository.config;
+
+import com.internship.tvseries.data.api.ConfigApi;
+import com.internship.tvseries.data.model.Configuration;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.function.Consumer;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class ConfigRetrofitRepository implements ConfigRepository {
```

Comment:
Let's change the name to this one to be more specific with what db source is used